cmd/v/internal/help/build.txt

```diff
@@ -0,0 +1,80 @@
+Usage: v [build flags] ['build'] <file.v|directory>
+
+This command compiles the given target, along with their dependencies, into an executable.
+
+When compiling packages, build ignores files that end in '_test.v'.
+
+When compiling a single main package, build writes the resulting executable to an output file
+named after the build target. ('v abc.v' and 'v abc/' both write either 'abc' or 'abc.exe')
+The '.exe' suffix is added when writing a Windows executable.
+By default, the executable is stored in the same directory as the compiled source code.
+
+The -o flag forces build to write the resulting executable or object to the d output file or directory,
+instead of the default behavior described in the last two paragraphs.
+
+You can put common options inside an environment variable named VFLAGS, so that
+you don't have to repeat them.
+
+You can set it like this: `export VFLAGS=\"-cc clang -debug\"` on *nix,
```

Comment:
there is no need to escape the " signs here anymore, because it is an ordinary .txt file